sql/mdl.h

```diff
@@ -396,12 +396,6 @@ struct MDL_key {
     SRID,
     ACL_CACHE,
     COLUMN_STATISTICS,
-<<<<<<< HEAD
-    BACKUP_LOCK, /* Oracle LOCK INSTANCE FOR BACKUP */
```

Comment:
Is this really not required any more?

Reply:
It was moved to a different location in the same file. We got the conflict because our added comment. I didn't readd the comment at the new place.